package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "titanium-sdk",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Please revert the version bump here as it's done automatically when releasing 

```suggestion
  "version": "1.0.0",
````